h/static/scripts/tests/controllers/dropdown-menu-controller-test.js

```diff
@@ -29,21 +29,32 @@ describe('DropdownMenuController', () => {
     return menuEl.classList.contains('is-open');
   }
 
+  function clickMenuLink() {
+    toggleEl.dispatchEvent(new Event('click', { cancelable: true }));
```

Comment:
Perhaps this is ok, and I recognise that you're mainly just extracting this into a common helper function here rather than actually writing this code, but direct use of `dispatchEvent` to simulate user actions always makes me nervous. Don't we use [`syn`](https://www.npmjs.com/package/syn) for stuff like this?